src/Aquifer.JsEngine/Tiptap/TiptapUtilities.cs

```diff
@@ -0,0 +1,41 @@
+﻿using JavaScriptEngineSwitcher.V8;
+
+namespace Aquifer.JsEngine.Tiptap;
+
+/// <summary>
+/// This class is purely a C# wrapper around TiptapUtilities.js.
+/// See also Aquifer.Tiptap where that file is generated.
+/// </summary>
+public sealed class TiptapUtilities : IDisposable
```

Comment:
I'd prefer that any classes like this have a public API surface area that only exposes the JS methods under the hood.